test/integration_test.py

```diff
@@ -1,15 +1,27 @@
-import unittest
 import pytest
 import sys
 import os
-from pathlib import Path
-import inspect
 
 spack_c2sm_path = os.path.join(os.path.dirname(os.path.abspath(__file__)),
                                '..')
 
 sys.path.append(os.path.normpath(spack_c2sm_path))
-from src import machine_name, log_with_spack, sanitized_filename
+from src import log_with_spack, sanitized_filename, all_packages, machine_name
+
+exclude_on_machine = {
+    'tsa': [
+        'cosmo',  # irrelevant
+        'flexpart-cosmo',  # No compatible compiler available
+    ]
+}
```

Comment:
The information that a test is excluded is quite far away from the actual test. This is weird because normally the pytest skip marker (`@pytest.mark.skip`) is at the top of a test.

Reply:
I moved it closer to the tests